gulpfile.js

```diff
@@ -1,17 +1,19 @@
 'use strict';
 
 const gulp   = require('gulp');
+const clean  = require('gulp-clean');
 const jade   = require('gulp-jade');
 const seq    = require('gulp-sequence');
 const shell  = require('gulp-shell');
+const rename = require('gulp-rename');
 const dotenv = require('dotenv');
 const fs     = require('fs');
 const KarmaServer  = require('karma').Server
 
 // Public tasks (serial)
 gulp.task('git:hooks:pre-commit', seq('jspm:unbundle'));
-gulp.task('postinstall',          seq('jspm:install', 'typings:install', 'git:hooks:install'));
-gulp.task('start',                seq('build:dev', 'env:watch', 'server:dev'));
+gulp.task('postinstall',          seq('jspm:install', 'typings:install', 'git:hooks:install', 'copy:deps'));
+gulp.task('start',                seq('build:dev', 'env:watch', 'copy:deps', 'server:dev'));
```

Comment:
just convenient so it happens as part of the regular workflow.
open to suggestions.